tracing/3401/runtime/common/src/apis.rs

```diff
@@ -115,17 +115,18 @@ macro_rules! impl_runtime_apis_plus_common {
 				}
 			}
 
+			#[cfg(not(feature = "disable-genesis-builder"))]
```

Comment:
Not part of the changes

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam-runtime-overrides/pull/125/commits/7c3d885dadc3cc2121df5c8f7f5623c5f07d2220